doc/classes/ButtonGroup.xml

```diff
@@ -25,7 +27,7 @@
 	</methods>
 	<members>
 		<member name="allow_unpress" type="bool" setter="set_allow_unpress" getter="is_allow_unpress" default="false">
-			If [code]true[/code], it is possible to unpress all buttons in this [ButtonGroup].
+			If [code]true[/code], it is possible to unpress all buttons in this [ButtonGroup]. If [code]false[/code], one button must always be pressed (except in the initial state where no button may be pressed).
```

Comment:
Are you able to toggle off the currently pressed button by pressing it again? If yes, the additional remark is not true.

Reply:
You can only toggle off the currently pressed button by pressing it again if [member allow_unpresss] is [code]true[/code].